src/quo2/components/settings/data_item/view.cljs

```diff
@@ -0,0 +1,91 @@
+(ns quo2.components.settings.data-item.view
+  (:require [quo2.theme :as quo.theme]
+            [react-native.core :as rn]
+            [quo2.components.icon :as icons]
+            [quo2.components.settings.data-item.style :as style]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [quo2.foundations.resources :as quo2.resources]
+            [react-native.fast-image :as fast-image]
+            [quo2.components.avatars.account-avatar.view :as account-avatar]
+            [quo2.components.list-items.preview-list :as preview-list]
+            [quo2.components.common.not-implemented :as not-implemented]))
+
+(defn- render-right-side
+  [label icon-right? icon-color]
+  [rn/view
+   {:style style/right-container}
+   (case label
+     :preview [preview-list/preview-list
+               {:type      :user
+                :size      24
+                :list-size 3}]
+     :graph   [text/text "graph"]
+     :none    nil)
+   (when icon-right?
+     [rn/view
+      {:style {:margin-left (if (or (= label :graph) (= label :none)) 12 8)}}
+      [icons/icon
+       (if (= :none label)
+         :i/copy
+         :i/chevron-right)
+       {:color icon-color
+        :size  20}]])])
+
+(defn- render-left-side
+  [theme title status size blur? description icon subtitle label icon-color]
+  [rn/view
+   [rn/view
```

Comment:
It's generally a good practice to break apart components based on the minimum amount of data they need. That way, Reagent can potentially skip the whole re-rendering of parts of the hiccup tree.

In this case, this `view` container and its children don't need to re-render if `status`, `blur?`, `description`, `icon`, `subtitle`, or `icon-color` change.

Reply:
Done